mobile/src/validation-schemas/task-note/task-note-create.validation-schema.ts

```diff
@@ -0,0 +1,15 @@
+import { getNameOf } from 'guruhub-shared/helpers/helpers';
+import Joi from 'joi';
+
+import { TaskNoteValidationMessage } from '~/common/enums/enums';
+import { TaskNoteFormRequestDto } from '~/common/types/types';
+
+const taskNoteCreate = Joi.object({
```

Comment:
Don't we have such validation schema in shared?

Reply:
Not found, for now.

Reply:
@ab3pc leave it as now, we don`t need this one in shared